docs/basic-features/data-fetching.md

```diff
@@ -489,6 +489,23 @@ This ensures that users always have a fast experience while preserving fast buil
 
 `fallback: true` will not _update_ generated pages, for that take a look at [Incremental Static Regeneration](#incremental-static-regeneration).
 
+#### `fallback: 'blocking'`
+
+If `fallback` is `'blocking'`, new paths not returned by `getStaticPaths` will wait for the HTML to be generated, similarly to SSR (hence why _blocking_), and cached for future requests so it only happens once per path.
+
+> You can think of `fallback: 'blocking'` as [`fallback: true`](#fallback-true), but instead of waiting on a [fallback page](#fallback-pages) while the page is generated, the user waits on a blank page for the generated HTML of the page.
```

Comment:
Maybe "nothing will be rendered" is better than "blank page":

```suggestion
> You can think of `fallback: 'blocking'` as [`fallback: true`](#fallback-true), but instead of rendering a [fallback page](#fallback-pages) while the page is generated, nothing will be rendered until the page generation is complete.
```